.github/workflows/mariner2.yml

```diff
@@ -0,0 +1,69 @@
+name: Build for mariner2
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l -eo pipefail {0}
+    container:
+      image: mcr.microsoft.com/cbl-mariner/base/core:2.0
+    steps:
+      - name: Install prereqs
+        run: |
+          tdnf install --noplugins --skipsignature -y -q ca-certificates git
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+      - name: Install dependencies
+        run: |
+          bash .install/mariner2.sh
+      - name: Get Redis
+        uses: actions/checkout@v4
+        with:
+          repository: redis/redis
+          ref: '7.2.1'
+          path: redis
+      - name: Build Redis
+        working-directory: redis
+        run: make install
+      - name: Build Conan library
+        run: |
+          make conan SHOW=1
+      - name: Build RedisSearch
+        run: |
+          make build REDISEARCH_MT_BUILD=0 SHOW=1
+      - name: Build VARIANT=mt-oss
+        run: |
+          make build VARIANT=mt-oss SHOW=1
+      - name: Build COORD=oss
+        run: |
+          make build COORD=oss SHOW=1
+      - name: Test
+        run: |
+          make test REDISEARCH_MT_BUILD=0 PLATFORM_MODE=1 QUICK=1
```

Comment:
Note that `REDISEARCH_MT_BUILD` should be an env var - I think you should od something similar to what's done in this PR - https://github.com/RediSearch/RediSearch/pull/3904/files#diff-46bb56ecb2a009e985ca9028a156382daedfda34c5239479a5168589f6ce3f6fR34